en/docs/install-and-setup/setup/api-controller/managing-rate-limiting-policies/throttle-policy-import-export.md

```diff
@@ -0,0 +1,133 @@
+
+You may encounter an error when an API or an Application is imported from an environment that has custom-made rate limiting policies to another environment. This happens if the API or the Application has subscribed to a rate limiting policy that is not in the destination environment. Importation fails to indicate the rate limiting policy is not supported.
+
+Before importing either the API or App to the destination environment, rate limiting policies are imported to the destination environment using a separate step. After rate limiting policies are imported to the destination environment, you can import the API/Application to the destination environment without any rate limiting policy support errors.
+
+## Operations
+
+This feature introduces 3 new operations
+
+1. Get rate limiting policies
+2. Export rate limiting policy
+3. Import rate limiting policy
+
+Alongside those new operations, 3 new Admin APIs and 3 new APICTL commands are introduced.
+
+## Get rate limiting policies
+
+Get rate limiting policies operation allows users to list the available rate limiting policies. It also allows the user to filter the rate limiting policies by rate limiting policy levels like Application, Subscription, Advanced, and Custom.
+
+Follow the instructions below to display a list of rate limiting API Policies in an environment using apictl:
+
+1.  Make sure that the WSO2 API Manager (WSO2 API-M) is started and that the relevant version of apictl is set up.   
+     For more information, see [Download and Initialize the apictl]({{base_path}}/install-and-setup/setup/api-controller/getting-started-with-wso2-api-controller/#download-and-initialize-the-apictl).
+2.  Log in to the WSO2 API-M in the environment by following the instructions in [Login to an Environment]({{base_path}}/install-and-setup/setup/api-controller/getting-started-with-wso2-api-controller/#login-to-an-environment).
+3.  Run the corresponding apictl command below to get (list) rate limiting Policies in an environment.
+
+    - **Command**
+        ```bash
+        apictl get policies rate-limiting -e &lt;env name>  -q &lt;query>
+
+        `-q (query)` - This only allows the user to filter out rate limiting policies by type
+        ```
+
+        !!! example
+
+            ```bash
+            apictl get  policies  rate-limiting  -e prod  -q type:sub
+            ```
+
+## Import a rate limiting policy
+
+Import rate limiting policy operation allows users to import exported rate limiting policy files to a different environment and it also has the capability to overwrite existing policies.
+
+Run the following apictl command to import a rate limiting policy.
+
+    -   **Command**
+
+        ```bash
+        import policy rate-limiting  -f &lt;file path>   -e &lt;env name>    -- update
```

Comment:
By the way decided to move this section to migration doc during the internal disucssion. So why we have it here? Let's be consistent with [1].

wrong command. apictl prefix is missing. Also, when writing the command let's not mix shorthand and long flags, i.e, -e, -f are in short form and --update is in long form, which is inconsistent.
We need one command with shorthand format, another command in long format as in [1].

[1] https://apim.docs.wso2.com/en/4.2.0/install-and-setup/setup/api-controller/managing-common-api-policies/managing-common-api-policies/#get-common-api-policies-in-an-environment